content/guides/composite_monitors.md

```diff
@@ -0,0 +1,161 @@
+
+![create-composite-5](/static/images/composite_monitors/create-composite-5.png)
+
+Since there's still a 'Group Matching Error' despite matching group-bys, we can assume that these monitors have no reporting sources in common.
+
+As soon as you select a compatible second monitor, the UI will:
+
+1. Populate the 'trigger when' field with the default trigger condition `a && b`,
+2. Show the current status of each individual monitor, and
+3. Show the current status of the proposed composite monitor, given the default trigger condition.
+
+![create-composite-3](/static/images/composite_monitors/create-composite-3.png)
+
+### Set a trigger condition
+
+In the 'trigger when' field, write your desired trigger condition using boolean operators, referring to individual monitors by their labels in the form (a, b, c, etc). You can use parentheses to control operator precedence and create more complex conditions. 
+
+These are all valid trigger conditions: 
+ 
+~~~
+!(a && b)
+a || b && !c
+(a || b) && (c || d)
+~~~
+
+Outside of a composite monitor's New Monitor and Edit forms (e.g. on its Status page), its individual monitors are known by their numeric IDs:
+
+![composite-status](/static/images/composite_monitors/composite-status.png)
+
+In the API, a composite monitor's trigger condition is called its query. While a non-composite monitor's query can encapsulate many things — a metric, tags, an `avg` function, a group-by, etc — a composite monitor's query is simply its trigger condition, defined in terms of its constituent monitors.
+
+For two non-composite monitors with the following queries:
+
+~~~
+"avg(last_1m):avg:system.mem.free{role:database} < 2147483648" # monitor ID: 1234
+"avg(last_1m):avg:system.cpu.system{role:database} > 50" # monitor ID: 5678
+~~~
+
+a composite monitor's query is simply `"1234 && 5678"`, `"!1234 || 5678"`, etc.
+
+
+### Write a notification message
+
+Write a notification message as you would with any other monitor, using the @-syntax (e.g. @you@example.com) to notify individuals or teams. In addition to your own message, notifications for the composite monitor will show the status of the individual monitors:
+
+~~~
+[Triggered] AAA Steve Test Composite 1
+
+Steve, your composite monitor has triggered! @steve@example.com
+
+Query: 51253 || 51719
+
+1 Alert | 3 OK
+
+* AAA Steve Test 1
+  ID: 51253
+  2 availability-zone groups
+  1 Alert | 3 OK
+
+* AAA Steve Test 2
+  ID: 51719
+  1 availability-zone group
+  4 OK
+
+The monitor was last triggered at Fri Mar 24 2017 15:56:21 EDT (28 secs ago)
+~~~
+
+After setting any other miscellaneous options, click 'Save'.
+
+## How composite monitors work
+
+In this section, we look at a few examples to show **how** and **when** Datadog will alert on composite monitors.
```

Comment:
Yeah. I think it's fine to be more specific. Composite users are generally more advanced, you have to be because you need to have created other monitors to make a composite query, so they should be used to seeing the alerting. And then we could keep the frequency section but use triggerable states (No data, alert, warn, unknown) instead of T or F and therefore show more precise examples as well?